metrics_ci_integration/test/jenkins/client/model/jenkins_build_test.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:ci_integration/jenkins/client/model/jenkins_build.dart';
+import 'package:test/test.dart';
+
+import '../../resources/jenkins_artifacts_resources.dart';
+
+void main() {
+  group("JenkinsBuild", () {
+    final timestamp = DateTime(2020);
+
+    final buildJson = {
+      'number': 1,
+      'duration': 10,
+      'timestamp': timestamp.millisecondsSinceEpoch,
+      'result': 'FAILED',
+      'url': 'url',
+      'artifacts': JenkinsArtifactsResources.artifactsJson,
+    };
+
+    final jenkinsBuild = JenkinsBuild(
+      number: 1,
+      duration: const Duration(seconds: 10),
+      timestamp: timestamp,
+      result: 'FAILED',
+      url: 'url',
+      artifacts: JenkinsArtifactsResources.artifacts,
+    );
```

Comment:
```suggestion
     final timestamp = DateTime(2020);
    const buildNumber = 1;
    const buildDuration = Duration(seconds: 10);
    const buildResult = 'FAILED';
    const buildUrl = 'url';

    final buildJson = {
      'number': buildNumber,
      'duration': buildDuration.inSeconds,
      'timestamp': timestamp.millisecondsSinceEpoch,
      'result': buildResult,
      'url': buildUrl,
      'artifacts': JenkinsArtifactsResources.artifactsJson,
    };

    final jenkinsBuild = JenkinsBuild(
      number: buildNumber,
      duration: buildDuration,
      timestamp: timestamp,
      result: buildResult,
      url: buildUrl,
      artifacts: JenkinsArtifactsResources.artifacts,
    );
```
Can we extract variables from jsons to make the tests more clean and dry ?